packages/devtools_app/lib/src/charts/treemap.dart

```diff
@@ -0,0 +1,493 @@
+import 'package:flutter/material.dart';
+
+import '../trees.dart';
+
+class Treemap extends StatefulWidget {
+  const Treemap({
+    @required this.rootNode,
+    @required this.levelsVisible,
+    @required this.width,
+    @required this.height,
+    @required this.onTap,
+  });
+
+  final TreemapNode rootNode;
+
+  // The depth of children visible from this treemap widget.
+  // For example, levelsVisible = 2
+  // ---------------
+  // |    l = 1    |
+  // |  ---------  |
+  // |  | l = 2 |  |
+  // |  |       |  |
+  // |  |       |  |
+  // |  ---------  |
+  // ---------------
```

Comment:
from this I am unclear whether `l = 1` are root's children or root. can you clarify?

Reply:
Updated comment to reflect that `l = 1` are root's children.